tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -1103,3 +1103,19 @@ def test_docstring(a: Field[[IDim], float64]) -> Field[[IDim], float64]:
         fieldop_with_docstring(a, out=a)
 
     test_docstring(a, offset_provider={})
+
+
+def test_set_values_at_call(fieldview_backend):
+    size = 10
+    a = np_as_located_field(IDim, JDim)(np.ones((size, size)))
+    b = np_as_located_field(IDim, JDim)(np.ones((size, size)))
+    c = np_as_located_field(IDim, JDim)(np.ones((size, size)))
+
+    @field_operator(backend=fieldview_backend)
+    def fieldop_vals(
+        a: Field[[IDim, JDim], float64], b: Field[[IDim, JDim], float64]
+    ) -> Field[[JDim, IDim], float64]:
+        d = a + b
+        return d
+
+    fieldop_vals(a, b, out=c, offset_provider={})
```

Comment:
```suggestion
def test_return_type_annotation():
    with pytest.raises(
        GTTypeError,
        match="Annotated return type does not match deduced return type.",
    ):
        @field_operator
        def foo(
            a: Field[[IDim], float64]
        ) -> Field[[JDim], float64]:
            return a
```
I simplified the test a little and removed the execution part.